app/models/notification.rb

```diff
@@ -79,7 +84,47 @@ def unarchive_if_updated
   def update_from_api_response(api_response, unarchive: false)
     attrs = Notification.attributes_from_api_response(api_response)
     self.attributes = attrs
+    update_subject
     unarchive_if_updated if unarchive
     save(touch: false) if changed?
   end
+
+  private
+
+  def download_subject
+    user.github_client.get(subject_url)
+  end
+
+  def update_subject
+    return unless Octobox.config.fetch_subject
+    if subject
+      # skip syncing if the notification was updated around the same time as subject
+      return if updated_at - subject.updated_at < 2.seconds
```

Comment:
Changed the logic here to be a little more intuitive to mean updated within 2 seconds.